extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -195,4 +206,37 @@ export class DeployConfigPage extends DacFxConfigPage {
 		this.databaseLoader.loading = false;
 		return true;
 	}
+	/*
+	Function that is used to change the radio button DOM based on whether databases exist or not.
+	*/
+	private async disableUpgradeRadioButton(upgradeRadioButton: azdata.RadioButtonComponent, newRadioButton: azdata.RadioButtonComponent): Promise<azdata.FormComponent> {
```

Comment:
the radio buttons don't need to be passed in as parameters since this function should have access to them

Reply:
Acknowledged, thank you!